tagger.go

```diff
@@ -41,7 +41,11 @@ type TagRule struct {
 	Tags  TagSet `yaml:"tags,omitempty" mapstructure:"tags"`
 }
 type TaggerConfig struct {
-	Instances []TagRule `yaml:"instances"`
+	Instances     []TagRule `yaml:"instances"`
+	Volumes       []TagRule `yaml:"volumes"`
+	NodeBalancers []TagRule `yaml:"nodebalancers"`
+	Domains       []TagRule `yaml:"domains"`
+	LKEClusters   []TagRule `yaml:"lkeclusters"`
```

Comment:
```suggestion
	LKEClusters   []TagRule `yaml:"lke_clusters"`
```
To support the config change